---
title: "Gene_linear_modelling"
output: odf_document
---

```{r Import Libraries, include=FALSE}
library(limma)
library(ggplot2)
```

```{r Load Data, include=TRUE}
load("~/Documents/Coding/Brain-Proteome/R/filtered_TMT_Summary_Data.RData")
```

```{r Additive Linear Model, include=TRUE}
nCols <- ncol(filtered_TMT_Summary_Data)
nGenes <- nrow(filtered_TMT_Summary_Data)

# additive linear model
addBatchFactor <- c(2,2,1,1,3,3,4,4)
addDiseaseGroup <- factor(rep(addBatchFactor,5))
addDesign <- model.matrix(~addDiseaseGroup + 0)
colnames(addDesign) <- c("Ctl", "AD", "PD","ADPD")

addFit <- lmFit(filtered_TMT_Summary_Data[,7:nCols], addDesign)

contrast.matrix <- makeContrasts(AD-Ctl, PD-Ctl, ADPD-Ctl, levels=addDesign)
addFit2 <- contrasts.fit(addFit, contrast.matrix)
addFit2 <- eBayes(addFit2)
topGenesAdd <- topTable(addFit2, coef=1, adjust="BH")

# interaction linear model
Ctl <-factor(rep(c(0,0,1,1,0,0,0,0),5))
AD <- factor(rep(c(1,1,0,0,0,0,1,1),5))
PD <- factor(rep(c(0,0,0,0,1,1,1,1),5))
intDesign <- model.matrix(~Ctl + AD + PD + AD:PD + 0)
intDesign <- intDesign[,2:5]
colnames(intDesign) <- c("Ctl", "AD", "PD","ADPD")

intFit <- lmFit(filtered_TMT_Summary_Data[,7:nCols], intDesign)

contrast.matrix <- makeContrasts(AD-Ctl, PD-Ctl, ADPD-Ctl, levels=intDesign)
intFit2 <- contrasts.fit(intFit, contrast.matrix)
intFit2 <- eBayes(intFit2)
topGenesInt <- topTable(intFit2, coef=1, adjust="BH")
```



```{r Volcano plot, include=TRUE}
# Generate Log-fold changes
adAddGenes <- topTable(addFit2, coef=1, number=1000000, sort.by="logFC")
pdAddGenes <- topTable(addFit2, coef=2, number=1000000, sort.by="logFC")
adpdAddGenes <- topTable(addFit2, coef=3, number=1000000, sort.by="logFC")

adIntGenes <- topTable(intFit2, coef=1, number=1000000, sort.by="logFC")
pdIntGenes <- topTable(intFit2, coef=2, number=1000000, sort.by="logFC")
adpdIntGenes <- topTable(intFit2, coef=3, number=1000000, sort.by="logFC")

# Add compute expressed genes threshold (TRUE is differentially expressed)
adAddGenes$threshold = as.factor(abs(adAddGenes$logFC) > 0.25 & adAddGenes$P.Value < 0.05/nGenes)
pdAddGenes$threshold = as.factor(abs(pdAddGenes$logFC) > 0.25 & pdAddGenes$P.Value < 0.05/nGenes)
adpdAddGenes$threshold = as.factor(abs(adpdAddGenes$logFC) > 0.25 & adpdAddGenes$P.Value < 0.05/nGenes)

adIntGenes$threshold = as.factor(abs(adIntGenes$logFC) > 0.25 & adIntGenes$P.Value < 0.05/nGenes)
pdIntGenes$threshold = as.factor(abs(pdIntGenes$logFC) > 0.25 & pdIntGenes$P.Value < 0.05/nGenes)
adpdIntGenes$threshold = as.factor(abs(adpdIntGenes$logFC) > 0.25 & adpdIntGenes$P.Value < 0.05/nGenes)

# Generate volcano plots using the additive model.
g1 = ggplot(data=adAddGenes, aes(x=logFC, y=-log10(P.Value), colour=threshold)) +
  geom_point(alpha=0.4, size=1.75) +
  theme(legend.position = "none") +
  xlim(c(-2.5, 2.5)) + ylim(c(0, 15)) +
  xlab("log2 fold change") + ylab("-log10 p-value") + ggtitle('Additive model: Ctrl vs. AD')
g1

g2 = ggplot(data=pdAddGenes, aes(x=logFC, y=-log10(P.Value), colour=threshold)) +
  geom_point(alpha=0.4, size=1.75) +
  theme(legend.position = "none") +
  xlim(c(-2.5, 2.5)) + ylim(c(0, 15)) +
  xlab("log2 fold change") + ylab("-log10 p-value") + ggtitle('Additive model: Ctrl vs. PD')
g2

g3 = ggplot(data=adpdAddGenes, aes(x=logFC, y=-log10(P.Value), colour=threshold)) +
  geom_point(alpha=0.4, size=1.75) +
  theme(legend.position = "none") +
  xlim(c(-2.5, 2.5)) + ylim(c(0, 15)) +
  xlab("log2 fold change") + ylab("-log10 p-value") + ggtitle('Additive model: Ctrl vs. ADPD')
g3

# Generate volcano plots using the interaction model.
g4 = ggplot(data=adIntGenes, aes(x=logFC, y=-log10(P.Value), colour=threshold)) +
  geom_point(alpha=0.4, size=1.75) +
  theme(legend.position = "none") +
  xlim(c(-2.5, 2.5)) + ylim(c(0, 15)) +
  xlab("log2 fold change") + ylab("-log10 p-value") + ggtitle('Interaction model: Ctrl vs. AD')
g4

g5 = ggplot(data=pdIntGenes, aes(x=logFC, y=-log10(P.Value), colour=threshold)) +
  geom_point(alpha=0.4, size=1.75) +
  theme(legend.position = "none") +
  xlim(c(-2.5, 2.5)) + ylim(c(0, 15)) +
  xlab("log2 fold change") + ylab("-log10 p-value") + ggtitle('Interaction model: Ctrl vs. PD')
g5

g6 = ggplot(data=adpdIntGenes, aes(x=logFC, y=-log10(P.Value), colour=threshold)) +
  geom_point(alpha=0.4, size=1.75) +
  theme(legend.position = "none") +
  xlim(c(-2.5, 2.5)) + ylim(c(0, 15)) +
  xlab("log2 fold change") + ylab("-log10 p-value") + ggtitle('Interaction model: Ctrl vs. ADPD')
g6
```
